### DISCUSSION TITLE
Tatsächliche Sonneneinstrahlung

### DISCUSSION THREAD

**Discussion by Heiko-L** [tbnobody/OpenDTU#1390] [Ideas]
Ist es bei den Hoymiles Wechselrichtern irgendwie möglich die "Reserveleistung" auszulesen wenn sie gedrosselt werden?

Also z.B. HMS-800 auf 10%, wären ja 80W, die Solarmodule würden aber im Moment auch 550W schaffen- diese 550W würde ich gerne als Wert auslesen können. Der HMS muss das ja, zumindest indirekt, erfassen können.

Der Wert "Einstrahlung" bei den einzelnen Strings skaliert ja mit dem Power-Limit, ist also nicht zu gebrauchen.

Vielen Dank,
Heiko

**Comment by DPO99** [comment]
Also ich glaube nicht, dass das geht. Wenn der WR den Strom erstmal so durchlässt (und das müsste er ja um zu schauen wie viel so kommt), dann muss das auch irgendwo hin. Entweder in einen Verbraucher oder er wandelt es halt in wärme um. M.E. funktioniert der WR eher wie eine Art Netzteil und lässt eben maximal X durch oder das was eben kommt (vielleicht täusch ich mich aber auch). Drosselung macht m.E. ja aber auch nur an einem Akku Sinn. Warum willst Du die Leistung denn überhaupt so weit runter drosseln, wenn du eh "nur" an Modulen hängst?

**Comment by Heiko-L** [comment]
Naja, der WR kennt ja seine aktuelle Ausgangsleistung und er weiß wie stark er den Eingang aktuell drosseln muss um auf diese Ausgangsleistung zu kommen.

Ich habe gerade eine Nulleinspeisung realisiert und jetzt ist mir aufgefallen, dass ich ja gar nicht mehr weiß wie viel Reserve überhaupt noch da ist ohne "kurz mal aufzudrehen".

**Comment by DPO99** [comment]
Da bin ich mir eben unsicher, ob er die wirklich kennt. M.E. ist das nicht notwendig um einen bestimmten (gedrosselten) Wert auszuspucken. Über den Sinn dieser Nulleinspeisung brauchen wir ja nicht zu diskutieren aber das ist für Dich doch eigentlich unerheblich, wie viel Rest du hast oder nicht? Ich meine was willst du denn machen wenn du 1500w bräuchtest...kommen ja eh nur maximal 600w oder illegale 800w. Und im Zweifel liefert er eben genau das, was du brauchst oder was noch übrig ist. Bei 600w würde ich persönlich jetzt auch nicht überlegen ob ich eine Extra Berechnung brauch um die Waschmaschine anzumachen...Denn abzüglich einer vorhandenen Grundlast bleiben von den maximal 600w ja eh nur 400 oder 500? übrig. 

Aber unterm Strich kann @tbnobody ja beantworten, ob so ein Wert auslesbar ist.

**Comment by Heiko-L** [comment]
Waschmaschine, Spülmaschine sind ja Sachen, die sowieso irgendwann laufen, da schaue ich nur, dass die jetzt halt bei Sonnenschein laufen. Wobei die auch nur beim Heizen viel Leistung brauchen, die Pumpen und Motoren sind ja recht genügsam.
Ich hätte aber noch einen Kühlschrank, der SuperCool oder SuperFreeze anbietet und somit ja auch in einem abgedrehten Sinne Solarenergie speichern kann. Auch ein Luftentfeuchter wäre zuschaltbar...
